### PR TITLE
feat: add security chapter to 2022/zh-CN directory

### PR DESCRIPTION
**Pull request by Levix**
Makes progress on #1067

### CODE REVIEW

**Comment by tunetheweb** [comment]
Converting to draft for now until the actual translation is completed.

**Comment by Levix** [comment]
Hi, @tunetheweb . Finished translation

File: src/config/2022.json [hunk 1]
```diff
@@ -1281,6 +1281,14 @@
...
       "github":"yokoka"
     },
+    "Levix":{
+      "name": "Zhiwei Li",
+      "teams":[
+        "translators"
+      ],
+      "avatar_url": "65117011",
+      "github":"Levix"
+    },
```
Comment (tunetheweb) [inline src/config/2022.json:1291]: @Levix can you confirm you're happy with this?

You can add Twitter, Personal website, or LinkedIn usernames too if you want.

**Review by tunetheweb — APPROVED** [review]
LGTM.

I've staged it here: https://20221126t153645-dot-webalmanac.uk.r.appspot.com/zh-CN/2022/security

@luckybai / @ibelem do you want to have a look at the actual translation?

File: src/config/2022.json (hunk 1 above)
Comment (Levix) [reply src/config/2022.json:1291]: Yeah.

**Comment by ibelem** [comment]
Many thanks and great work, @Levix! The overall translation is in very high quality, it's hard to catch Chinese grammar issue and typos. 

I understand that there is no unified standard in the Chinese world for 柱状图/条形图, but the answers in multiple JS Chart frameworks and Q&A show that vertical columns to display data are usually called 柱状图, and horizontal columns to display data are more called 条形图。The two situations are reversed in your translation, please check if you can update them.

https://www.highcharts.com.cn/docs/column-and-bar-charts
https://www.zhihu.com/question/56819190

File: src/content/zh-CN/2022/security.md [hunk 2]
```diff
@@ -0,0 +1,1280 @@
...
+  caption="加密货币市场份额（移动端）",
+  description="饼状图显示 CoinImp 占 60.4% 的市场份额，Coinhive 占 15.6%，Minero 占 12.4%，JSECoin 占 6.8%，其他约占 4.8%。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSPHK3G2Ir-ys_oTrrhugqxV0aOSj3y5d1lANQ54GdaQtIHrzXIjQQGEpIdT_mQvxTrMtpd0Hn30zhF/pubchart?oid=1570078079&format=interactive",
+  sheets_gid="280179954",
+  sql_file="cryptominer_share.sql"
+) }}
+
+这些结果表明，加密劫持仍然是一个严重的攻击载体，每年根据新出现的漏洞使其使用量激增。因此，为了减轻这一领域的风险，仍然需要适当的努力。
+
+注意，并非所有这些网站都受到了感染，网站运营商也可能部署这种技术(而不是展示广告)来资助他们的网站，但这种技术的使用也在技术上、法律上和伦理上被大量讨论。
```
Comment (ibelem) [inline src/content/zh-CN/2022/security.md:1216]: /(而不是展示广告)/（而不是展示广告）/g

**Review by ibelem — COMMENTED** [review]
@tunetheweb I've completed the review of this round. Thanks!

File: src/content/zh-CN/2022/security.md [hunk 3]
```diff
@@ -0,0 +1,1280 @@
...
+}
+```
+
+这在很大程度上被称为点击劫持攻击，是嵌入在 iframe 中的许多其他安全风险之一（双关语）。
+
+为了减轻这些担忧，HTML 规范（第 5 版）引入了 `sandbox` 属性，可以应用于 iframe 元素，它充当一个白名单列表，如果保持为空，它基本上不启用 iframe 元素中的任何功能，这将导致无法访问弹出窗口等页面交互功能，没有运行 JavaScript 代码的权限，也无法访问 cookies。
+
+{{ figure_markup(
+  image="prevalence-of-sandbox-directives-on-frames.png",
+  caption="框架上 sandbox 指令的普遍性",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md:729]: Then just keep frames ?

File: src/content/zh-CN/2022/security.md [hunk 4]
```diff
@@ -0,0 +1,1280 @@
...
+如果 cookie 设置不当，可能容易遭受许多不同形式的攻击，例如<a hreflang="en" href="https://owasp.org/www-community/attacks/Session_hijacking_attack">会话劫持</a>、<a hreflang="en" href="https://owasp.org/www-community/attacks/csrf">跨站请求伪造（CSRF）</a>、<a hreflang="en" href="https://owasp.org/www-project-web-security-testing-guide/v41/4-Web_Application_Security_Testing/11-Client_Side_Testing/13-Testing_for_Cross_Site_Script_Inclusion">跨站脚本包含（XSSI）</a>和其他各种<a hreflang="en" href="https://xsleaks.dev/">跨站泄漏</a>漏洞。
+
+### Cookie 属性
+
+为了防御上述威胁，开发人员可以在 cookie 中使用 3 个不同的属性：`HttpOnly`、`Secure` 和 `SameSite`。`Secure` 属性类似于 `HSTS` 标头，因为它也确保 cookie 必须通过 HTTPS 发送，防止<a hreflang="en" href="https://owasp.org/www-community/attacks/Manipulator-in-the-middle_attack">中间人攻击</a>。`HttpOnly` 确保 cookie 不能被任何 JavaScript 代码访问，防止<a hreflang="en" href="https://owasp.org/www-community/attacks/xss/">跨站脚本</a>攻击。
+
+{{ figure_markup(
+  image="httponly-secure-samesite-cookie-usage.png",
+  caption="Cookie 属性 (桌面端)",
+  description="桌面端网站上使用 cookie 属性条形图，按第一方和第三方 cookie 划分。第一方 cookie 使用 `HttpOnly` 的比例为 36%，`Secure` 的比例为 37%，`SameSite` 的比例为 45%，而第三方 cookie 使用`HttpOnly` 的比例为 21%，`Secure` 的比例为 90%，`SameSite` 的比例为 88%。",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md:224]: "甲方" feels a bit strange.

File: src/content/zh-CN/2022/security.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ibelem) [reply src/content/zh-CN/2022/security.md:729]: Personally I prefer to keep "frames"

File: src/content/zh-CN/2022/security.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ibelem) [reply src/content/zh-CN/2022/security.md:224]: I know, but 第一方 is also weird... Not sure if any better tanslation.

File: src/content/zh-CN/2022/security.md [hunk 5]
```diff
@@ -0,0 +1,1280 @@
...
+lirantal_bio: <a hreflang="en" href="https://www.lirantal.com/">Liran Tal</a> 因其开源和 JavaScript 安全倡议而闻名，并在 Node.js 安全方面的工作获得了OpenJS 基金会的安全探路者奖项，是国际公认的 <a hreflang="en" href="https://stars.github.com/profiles/lirantal/">GitHub Star</a>，他是 JavaScript 社区获奖的软件开发人员、安全研究员和开源倡导者。他对开发者安全教育的贡献包括领导 OWASP 项目，建立供应链安全工具，参与 CNCF 和 OpenSSF 计划，并撰写了 O'Reilly 的 《Serverless Security》 等书籍，他领导着 Snyk.io 开发者宣传团队，并以赋予开发者更好的应用安全技能为使命。
+clarkio_bio: Brian 是一位在应用安全方面有深入经验的 Web 开发者，通过在 Snyk.io 担任开发者大使的工作，帮助开发者构建安全的 Web 应用。虽然他有全栈项目的工作经验，但他的重点是后端服务、API 和开发者工具。Brian 喜欢向开发人员传授他从整个职业生涯的成功和失败中学到的东西。你可以在他<a hreflang="en" href="https://clarkio.live">每周的直播</a>中或在他的 <a hreflang="en" href="https://www.pluralsight.com/authors/brian-clark">Pluralsight 课程</a>中找到他正在做的事情。
+results: https://docs.google.com/spreadsheets/d/1cwJ43NL2IN2PxJa5oiOoJCRkSh566XE_k9uHnGJdWeg/
+featured_quote: 人们的个人信息以及生活的各个方面每天都在变得越来越数字化，这使得安全和隐私变得极为重要，网站有责任采用最佳的安全实践来确保其用户隐私受到保护。
+featured_stat_1: +14%
+featured_stat_label_1: Increase in adoption for Content Security Policy
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: OK. fixed.

File: src/content/zh-CN/2022/security.md [hunk 6]
```diff
@@ -0,0 +1,1280 @@
...
+featured_stat_2: 428
+featured_stat_label_2: Websites using the `Clear-Site-Data` header
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 7]
```diff
@@ -0,0 +1,1280 @@
...
+featured_stat_3: +85%
+featured_stat_label_3: Increase in adoption of Permissions Policy
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 8]
```diff
@@ -0,0 +1,1280 @@
...
+
+{{ figure_markup(
+  content="25%",
+  caption="具有 HSTS 头的移动端请求",
+  classes="big-number",
+  sheets_gid="822440544",
+  sql_file="hsts_attributes.sql",
+) }}
+
+25% 的移动端响应和 28% 的桌面相应携带 HSTS 头。
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 9]
```diff
@@ -0,0 +1,1280 @@
...
+
+HSTS 是使用 `Strict-Transport-Security` 请求头来设置的，它可以有三种不同的指令：`max-age`、`includeSubDomains` 和 `preload`。 `max-age` 表示浏览器应该记住只使用 HTTPS 访问网站的时间，以秒为单位，是该请求头的一个强制性指令。
+
+{{ figure_markup(
+  image="hsts-directives-usage.png",
+  caption="不同 HSTS 指令的使用情况",
+  description="不同 HSTS 指令使用情况百分比条形图。19% 的网站在移动和桌面端都使用了 `preload`，`includeSubdomain` 在 37% 的桌面网站和 34% 的移动网站中被使用，Zero `max-age`用于 6% 的桌面网站和 5% 的移动网站，94% 的桌面网站和 95% 的移动网站设置了有效的 `max-age`。",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 10]
```diff
@@ -0,0 +1,1280 @@
...
+  sheets_gid="1093490384",
+  sql_file="https_request_over_time.sql",
+) }}
+
+桌面端 94% 的请求和移动端 93% 的请求都是通过 HTTPS 发送的，所有主流浏览器现在都有一个 <a hreflang="en" href="https://support.mozilla.org/en-US/kb/https-only-prefs">HTTPS-only 模式</a>，如果一个网站使用 HTTP 而非 HTTPS，则会显示警告。
+
+{{ figure_markup(
+  image="https-usage-by-site.png",
+  caption="网站的 HTTPS 使用率",
+  description="柱状图显示 89% 的桌面端站点使用 HTTPS，剩下的 11% 使用 HTTP，而 85% 的移动端站点使用 HTTPS，剩下的 15% 使用 HTTP。",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 11]
```diff
@@ -0,0 +1,1280 @@
...
+
+### 协议版本
+
+重要的是，不仅要使用 HTTPS，而且要使用最新的 TLS 版本，<a hreflang="en" href="https://datatracker.ietf.org/doc/rfc8996/">TLS 工作组</a>已经废弃了 TLS v1.0 和 v1.1，因为它们存在多个缺陷。自从去年的章节以来，Firefox <a hreflang="en" href="https://support.mozilla.org/en-US/kb/secure-connection-failed-firefox-did-not-connect#w_tls-version-unsupported">现在已经更新了它的用户界面</a>，在 Firefox 97 版的错误页面中，启用 TLS 1.0 和 1.1 的选项已经被删除。Chrome 也从 98 版本开始<a hreflang="en" href="https://chromestatus.com/feature/5759116003770368">不再允许绕过</a> TLS 1.0 和 1.1 所显示的错误页面。
+
+IETF 于 2018 年 8 月发布的 TLS v1.3 是最新版本，相较于 TLS v1.2 版本它<a hreflang="en" href="https://www.cloudflare.com/en-in/learning/ssl/why-use-tls-1.3/">更快而且更安全</a>，TLS v1.2 中的许多主要漏洞都与旧加密算法有关，而 TLS v1.3 删除了这些算法。
+
+{{ figure_markup(
+  image="tls-version-by-site.png",
+  caption="网站的 TLS 版本使用分布",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md:65]: fixed

File: src/content/zh-CN/2022/security.md [hunk 12]
```diff
@@ -0,0 +1,1280 @@
...
+在上图中，我们看到 70% 的移动网页和 67% 的桌面网页是通过 TLSv1.3 提供的，这比去年增加了约 7%。因此，我们看到一些从使用 TLS v1.2 到 TLS v1.3 的持续转变。
+
+### 密码套件（Cipher suites）
+
+<a hreflang="en" href="https://learn.microsoft.com/en-au/windows/win32/secauthn/cipher-suites-in-schannel">密码套件</a>是一组加密算法，客户和服务器在开始使用 TLS 进行安全通信之前必须达成一致。
+
+{{ figure_markup(
+  image="distribution-of-cipher-suites.png",
+  caption="密码套件的分布",
+  description="堆叠柱状图显示了各设备使用的密码套件，`AES_128_GCM` 是最常见的，被 79% 的桌面端网站和 79% 的移动端网站使用，`AES_256_GCM` 被 19% 的桌面端网站和 18% 的移动端网站使用，`AES_256_CBC` 被 2% 的桌面端网站和 1% 的移动端网站使用，`CHACHA20_POLY1305` 分别被 1% 和 1% 的网站使用，`AES_128_CBC` 分别被 0% 和 0% 使用。",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 13]
```diff
@@ -0,0 +1,1280 @@
...
+  sql_file="tls_cipher_suite.sql"
+  )
+}}
+
+现代 [Galois/Counter Mode (GCM)](https://en.wikipedia.org/wiki/Galois/Counter_Mode) 密码模式被认为是更安全的，因为它们不容易受到<a hreflang="en" href="https://blog.qualys.com/product-tech/2019/04/22/zombie-poodle-and-goldendoodle-vulnerabilities">填充攻击（padding attacks）</a>。TLS v1.3 <a hreflang="en" href="https://datatracker.ietf.org/doc/html/rfc8446#page-133">只支持 GCM 和其他现代分组密码模式</a>，使其更加安全，它还消除了<a hreflang="en" href="https://go.dev/blog/tls-cipher-suites">密码套件排序</a>的麻烦。决定密码套件使用情况的另一个因素是用于加密和解密的密钥大小，我们看到 128 位的密钥仍然被广泛使用，因此，我们看到与去年的图表对比没有什么不同，`AES_128_GCM` 仍然是使用最多的密码套件，使用率为 79%。
+
+{{ figure_markup(
+  image="forward-secrecy-support.png",
+  caption="前向保密（Forward secrecy）的使用情况",
+  description="柱状图显示，97% 的移动端和桌面端请求都使用了前向保密。",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 14]
```diff
@@ -0,0 +1,1280 @@
...
+  <figcaption>
+    {{ figure_link(
+      caption="iframes 上 `allow` 指令的普遍性",
+      sheets_gid="1848560369",
+      sql_file="iframe_allow_directives.sql"
+    ) }}
+  </figcaption>
+</figure>
+
+有趣的是，第 11、12 和 13 位的移动端指令没有进入到上述名单，它们是 `vr`（占 6%），`payment`（占 2%），`web-share`（占 1%），它们也许说明了我们在虚拟现实（如果你愿意，也可以称之为元宇宙）、在线支付和金融科技行业中看到的日益增长的趋势。最后，这似乎表明了基于 web 共享得到了更好的支持，这可能是由于过去几年在家办公（指新冠疫情期间）的兴起。
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 15]
```diff
@@ -0,0 +1,1280 @@
...
+```
+
+这在很大程度上被称为点击劫持攻击，是嵌入在 iframe 中的许多其他安全风险之一（双关语）。
+
+为了减轻这些担忧，HTML 规范（第 5 版）引入了 `sandbox` 属性，可以应用于 iframe 元素，它充当一个白名单列表，如果保持为空，它基本上不启用 iframe 元素中的任何功能，这将导致无法访问弹出窗口等页面交互功能，没有运行 JavaScript 代码的权限，也无法访问 cookies。
+
+{{ figure_markup(
+  image="prevalence-of-sandbox-directives-on-frames.png",
+  caption="框架上 sandbox 指令的普遍性",
+  description="图表显示 iframes 中 sandbox 指令的流行程度。`allow-scrips` 和 `allow-same-origin` 是使用最多的指令，几乎 100% 具有 `sandbox` 属性的 iframes 都使用了这些指令。`allow-popups` 出现在 82% 的桌面端和 85% 的移动端框架中；`allow-forms` 出现在 81% 的桌面端和 84% 的移动端框架中；`allow-popups-to-escape-sandbox` 出现在 80% 的桌面端和 83% 的移动端框架中； `allow-top-navigation-by-user-activation` 出现在 59% 的桌面端和 61% 的移动端框架中；`allow-top-navigation` 出现在 21% 的桌面端和 22% 的移动端框架中，`allow-modals` 在桌面端占 20%，在移动端占 21%。",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 16]
```diff
@@ -0,0 +1,1280 @@
...
+)
+}}
+
+防止点击劫持攻击的常见方法之一是防止浏览器在框架中加载网站，人们可以在 CSP 标头中使用 [`frame-ancestors`](https://developer.mozilla.org//docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors) 指令来限制其他域将页面内容嵌入框架中。我们发现 53% 设置了 CSP 的 移动网站都有 `frame-ancestor` 指令，这是第二大使用的 CSP 指令，这有助于防止点击劫持攻击。将 `frame-ancestors` 指令的值设置为 `none` 或 `self` 是最安全的，`none` 不允许任何域名框住内容，而 `self` 只允许原始域名框住内容。我们发现，在有 CSP 标头的移动端网站中，有 8% 的网站只设置了 `frame-ancestors 'self'`，是 CSP 标头的第三大常见值。
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Keyword</th>
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 17]
```diff
@@ -0,0 +1,1280 @@
...
+与去年相比，数据没有太大变化。`CryptoGetRandomValues` 仍然是采用最多的特性（尽管自去年以来它的使用量下降了大约 1%），因为它用于生成强伪随机数，它的高使用率是由于在诸如谷歌 Analytics 等常见服务中被使用。
+
+### 机器人保护服务（Bot protection services）
+
+今天的互联网充满了机器人，因此恶意机器人攻击不断上升。根据 Imperva <a hreflang="en" href="https://www.imperva.com/resources/reports/2022-Imperva-Bad-Bot-Report.pdf">2022 年恶意机器人报告</a>， 27.7% 的互联网流量是由恶意机器人造成的。恶意机器人是那些试图抓取数据并利用它的机器人，根据该报告，2021 年底恶意机器人攻击激增，可能是因为 log4j 漏洞可被机器人利用。
+
+{{ figure_markup(
+  image="bot-protection-service-usage.png",
+  caption="供应商使用机器人保护服务的情况",
+  description="条形图显示了机器人保护服务的使用情况。19.9% 的桌面端网站和 18.4% 的移动端网站使用 reCaptcha；7.9% 的桌面端网站和 6.1% 的移动端网站使用 Cloudflare 机器人管理；有 0.6% 的桌面端网站和 0.5% 的移动端网站使用 Akamai 机器人管理；有 0.4% 的桌面端网站和 0.3% 的移动端网站使用了 Imperva；其他机器人保护服务有 0.4% 的网站在桌面端使用，0.3% 的网站在移动端使用。",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 18]
```diff
@@ -0,0 +1,1280 @@
...
+- 知名度：广受欢迎的网站可能比鲜为人知的网站面临更多有针对性的攻击
+
+### 网站地理位置
+
+网站地理位置、网站开发人员的所在地或网站的托管地往往会对安全功能的采用产生影响，这可能是因为开发人员之间的意识不同，但也可能是因为国家的法律强制采用某些安全实践。
+
+{{ figure_markup(
+  image="adoption-of-https-per-country.png",
+  caption="每个国家启用 HTTPS 的情况",
+  description="条形图显示了与不同国家有关的网站启用 HTTPS 的百分比。新西兰、尼日利亚、澳大利亚、挪威、瑞士、丹麦、爱尔兰、南非、荷兰和加拿大依次为 95% 至 93%。在另一端，越南、塞尔维亚、俄罗斯联邦、波兰、伊朗、印度尼西亚、台湾、泰国、韩国和日本为 85% 至 81%。",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 19]
```diff
@@ -0,0 +1,1280 @@
...
+- 技术层面：在某些技术栈中采用安全特性可能更容易，或者某些供应商可能在默认情况下启用安全特性
+- 知名度：广受欢迎的网站可能比鲜为人知的网站面临更多有针对性的攻击
+
+### 网站地理位置
+
+网站地理位置、网站开发人员的所在地或网站的托管地往往会对安全功能的采用产生影响，这可能是因为开发人员之间的意识不同，但也可能是因为国家的法律强制采用某些安全实践。
+
+{{ figure_markup(
+  image="adoption-of-https-per-country.png",
+  caption="每个国家启用 HTTPS 的情况",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md [hunk 20]
```diff
@@ -0,0 +1,1280 @@
...
+上面我们看到一些常见的 CMS 和博客网站，我们看到的一个共同模式是，像 Wix、Squarespace 和 Medium 这样很少提供定制服务，更注重内容编辑的网站，它们默认有基本的安全功能，如 `Strict-Transport-Security`。像 Wagtail、Plone 和 Drupal 这样的内容管理系统只采用了非常少的安全功能，因为它们经常被开发人员用来建立网站，因此增加安全功能的责任更多地落在了开发人员身上。我们还看到，使用谷歌站点的网站在默认情况下具有许多安全特性。
+
+### 网站知名度
+
+访问者多的网站可能更容易受到有针对性的攻击，因为有更多的用户拥有潜在的敏感数据来吸引攻击者。因此，可以预期的是浏览量大的网站为了保护用户，会在安全方面投入更多资金。
+
+{{ figure_markup(
+  image="prevalence-of-headers-in-sites-by-rank.png",
+  caption="第一方环境中按等级设置安全标头的流行度",
+  description="条柱状图显示，在排名前 1000 位的网站中，55.9% 设置了 XFO，56.8% 设置了 HSTS，27% 设置了 CSP 标头；在排名前 10,000 的网站中，51.3% 设置了 XFO，44.5% 设置了 HSTS，20.6% 设置了 CSP 标头；在前 100,000 名中，48.1% 设置了 XFO，38% 设置了 HSTS，17.8% 设置了 CSP 标头；在排名前 100 万的网站中，41.9% 设置了 XFO，31.1% 设置了 HSTS，16.4% 设置了 CSP 标头；在所有网站中，31% 设置了 XFO，25.7% 设置了 HSTS，14.6% 设置了 CSP。",
```
[the remark this answers is not in the document]
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md (hunk 2 above)
Comment (Levix) [reply src/content/zh-CN/2022/security.md]: fixed

File: src/content/zh-CN/2022/security.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ibelem) [reply src/content/zh-CN/2022/security.md]: I am OK to keep it.

**Comment by ibelem** [comment]
Thanks @Levix , no more translation review comment on my side. @tunetheweb

**Comment by tunetheweb** [comment]
Thanks both - great collaboration!

**Comment by tunetheweb** [comment]
And it's live!: https://almanac.httparchive.org/zh-CN/2022/security

**Pull request merged**